editions/tw5.com/tiddlers/String Operators.tid

```diff
@@ -0,0 +1,8 @@
+created: 20230803054456864
+modified: 20230803054957952
+tags: Filters [[Filter Operators]]
+title: String Operators
+
+String operators are [[filter operators|Filter Operators]] that interact with strings.
+
+<<list-links "[tag[String Operators]]">>
```

Comment:
I would prefer: `<<list-links "[tag[String Operators]]" class:"multi-columns">>` which looks like this

![image](https://github.com/Jermolene/TiddlyWiki5/assets/374655/76753a1a-7381-48f9-bad7-cdc107b77576)

Reply:
only for those with many items in the list

Reply:
👍 Agreed and done!